test/recipes/81-test_cmp_cli.t

```diff
@@ -24,15 +24,19 @@ BEGIN {
 use lib srctop_dir('Configurations');
 use lib bldtop_dir('.');
 use platform;
+plan skip_all => "These tests are not supported in a fuzz build"
+    if config('options') =~ /-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION/;
 
 plan skip_all => "These tests are not supported in a no-cmp build"
     if disabled("cmp");
 plan skip_all => "These tests are not supported in a no-ec build"
     if disabled("ec");
-plan skip_all => "These tests are not supported in a fuzz build"
-    if config('options') =~ /-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION/;
-plan skip_all => "Tests involving server not available on Windows or VMS"
+plan skip_all => "Tests involving CMP server not available on Windows or VMS"
     if $^O =~ /^(VMS|MSWin32)$/;
+plan skip_all => "Tests involving CMP server require 'lsof' command"
+    unless `which lsof`;
+plan skip_all => "Tests involving CMP server require 'kill' command"
+    unless `which kill`; # this should also exclude Solaris
```

Comment:
I've experimented with using `type` instead of `which` and using `system()` instead of the backticks, but then the discrimination between existing and non-existing shell commands did not work properly.

I'm pretty confident that  `which` is also a built-in/available command in many (or even all usual) shells.
And I meanwhile got the confirmation https://github.com/openssl/openssl/issues/12324#issuecomment-657921753 that this PR works fine on Solaris as expected.

Reply:
Interesting, thanks for testing (that does not match my recollection/experience).  It seems reasonable to place more value on the actual test results than on theoretical ideas about how things "should work".

FWIW, I see:

```
bkaduk@sea-lp9yo:~$ echo $SHELL
/bin/bash
bkaduk@sea-lp9yo:~$ type type
type is a shell builtin
bkaduk@sea-lp9yo:~$ echo $?
0
bkaduk@sea-lp9yo:~$ type quxx
bash: type: quxx: not found
bkaduk@sea-lp9yo:~$ echo $?
1
bkaduk@sea-lp9yo:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic
```

`/bin/sh` and `/bin/csh` on my FreeBSD machine are similar (but return 127 instead of 1 for the "not found" case).